stable/firehose/templates/configmap-firehose.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "firehose.fullname" . }}-config
+  namespace: {{ .Values.namespace }}
```

Comment:
I think we should `Release.Namespace` instead of taking it from values. 
`Release.Namespace` is a special variable available and passed as a direct argument under helm command